packages/govern-subgraph/schema.graphql

```diff
@@ -1,22 +1,55 @@
-type Govern @entity {
+type ERC3000Registry @entity {
   id: ID!
   address: Bytes!
+  count: Int!
+  entries: [Entry!]
+}
+
+type Entry @entity {
+  id: ID!
   name: String!
-  queue: OptimisticQueue! @derivedFrom(field: "govern")
-  roles: [Role]!
+  dao: Govern!
+  queue: OptimisticQueue!
+  metadata: Bytes
+}
+
+type Govern @entity {
```

Comment:
I change quite a bit the `Govern` entity to focus on the executions it does as it's the main responsability of the Govern smartcontract. I also remove the link with the `ERC3000` (OptimisticQueue) entities to simplify and focus on the main functionality of the executions.

At one point I thought of including a `queues` attribute to keep track of every queue associated with the particular Govern. But I'm nos super sure how useful that might be.

Reply:
Yep, gotcha. I'd definitely keep the link with the `OptimisticQueue` as right now the thought process in the contracts is to have a Queue associated with a Govern entity, but of course this might change. We can keep it this way though if we're gonna use the `ERC3000Registry` to lookup DAOs, which _should_ always retrieve both the Govern entity and the Queue; if not, we could effectively have "headless" DAOs where we can only find its Govern entity, but not its Queue.

On the other side, I do like the change 😄 but I'm hesitant about the `dao` field as I don't know if we want to be so specific about the naming. Would love to hear @izqui thoughts on this

Reply:
Cool, yes I think that's the way to go, have `Govern` and `OptimisticQueue` linked at the `ERC3000Registry` entity and maybe a better name for the pair would be games as we can consider that each queue would be like the dao is playing a particular optimistic game?

On the naming I agree. I leave dao for now until we think a better name.